decidim-verifications/app/commands/decidim/verifications/id_documents/admin/confirm_user_offline_authorization.rb

```diff
@@ -58,8 +58,8 @@ def authorization
                                .new(organization: form.current_organization, name: "id_documents", granted: false)
                                .query
                                .where("verification_metadata->'rejected' IS NULL")
```

Comment:
How is this working if verification_metadata is encrypted?

Reply:
NULL values are not encrypted. If the JSON field is empty, it will appear as NULL in the database.

Reply:
ah, ok, my fault, it's only the values that are encrypted, not the whole hash itself, so the keys are still readable and "queryable",